include/zephyr/kernel/smp.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2023 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_KERNEL_SMP_H_
+#define ZEPHYR_INCLUDE_KERNEL_SMP_H_
+
+/**
+ * @brief Start one CPU.
+ *
+ * This starts (powers up/starts up) a CPU that has stopped
+ * or powered down. After the CPU is started, the scheduler
+ * will run, and the CPU will participate in execution of
+ * threads.
```

Comment:
"one" -> "a"

I think this opening paragraph needs a little rework. Maybe something like the following (which is itself incomplete) ...

> This routine is used to manually start the CPU specified by _id_. It may be called to restart a CPU that had been stopped or powered down, as well as _some other scenario_. After the CPU has finished initialization, the CPU will be ready to participate in thread scheduling and execution.

Reply:
Updated the working.